tests/python/pants_test/engine/test_fs.py

```diff
@@ -74,7 +74,7 @@ def assert_digest(self, filespecs_or_globs, expected_files):
       scheduler = self.mk_scheduler(rules=create_fs_rules(), project_tree=project_tree)
       result = self.execute(scheduler, Snapshot, self.specs(filespecs_or_globs))[0]
       # Confirm all expected files were digested.
-      self.assertEqual(set(expected_files), set(f.path for f in result.files))
+      self.assertEqual({expected_files}, {f.path for f in result.files})
```

Comment:
`set(expected_files)` -> `{expected_files}` might not be right. I think it'll mean that expected_files will be an element of the set, rather than copied into one.

Reply:
You're totally right. Thanks for the close review!